src/index.js

```diff
@@ -83,25 +86,57 @@ function replaceGithubLink(message, markdownLink) {
   );
 }
 
+/**
+ * Escape chars in raw string which should not be processed as marked text
+ *
+ * List of chars to be transcoded
+ * https://core.telegram.org/bots/api#html-style
+ *
+ * @param {string} message - string to be processed
+ * @returns {string}
+ */
+function escapeChars(message) {
+  message = message.replaceAll('<', '&lt;');
+  message = message.replaceAll('>', '&gt;');
+  message = message.replaceAll('&', '&amp;');
+
+  return message;
+}
+
+/**
+ * Parse github link via jonschlinkert/parse-github-url module
+ *
+ * https://github.com/jonschlinkert/parse-github-url
+ *
+ * @param {string} url - any github link (to pr or issue for example)
+ * @returns {string} - HTML marked link to repo
+ */
+function createTaskBadge(url) {
+  const repoInfo = parseGithubUrl(url);
+
+  return `<a href="https://github.com/${repoInfo.repo}"><b>${repoInfo.name}</b></a>`;
+}
+
 /**
  * parse the response of GraphQL query for pull request.
  *
  * @param {object} content - response of GraphQL API.
  * @returns {string} - parsed message.
  */
 function pullRequestParser(content) {
-  let parsedTask = `[${content.title.replace(/[[\]']+/g, '')}](${content.url}) @${content.author.login}`;
-
-  content.reviewRequests.nodes.forEach((node) => {
-    if (node.requestedReviewer.login) {
-      parsedTask += `@${node.requestedReviewer.login}`;
-    }
-  });
-  content.assignees.nodes.forEach((node) => {
-    if (node.login) {
-      parsedTask += `@${node.login}`;
-    }
-  });
+  let parsedTask = `${createTaskBadge(content.url)}: <a href="${content.url}">${escapeChars(content.title)}</a> @${content.author.login}`;
+
+  // content.reviewRequests.nodes.forEach((node) => {
+  //   if (node.requestedReviewer.login) {
+  //     parsedTask += `@${node.requestedReviewer.login}`;
+  //   }
+  // });
+  //
+  // content.assignees.nodes.forEach((node) => {
+  //   if (node.login) {
+  //     parsedTask += `@${node.login}`;
+  //   }
+  // });
```

Comment:
Why we removed the requested reviewer and assignees

Reply:
we can discuss it too. i think it is now necessary to duplicate link to pr. u can see repo name and request name and if it is project which u working for u should check it